packages/cozy-harvest-lib/src/services/budget-insight.js

```diff
@@ -363,7 +363,7 @@ export const onBIAccountCreation = async ({
   return account
 }
 
-export const sendTwoFaCode = (flow, code) => {
+export const sendTwoFACode = (flow, code) => {
```

Comment:
I think test will fail since in the spec we have
```
   sendTwoFaCode
} from './budget-insight' 
```

Reply:
Thanks, I fixed unit tests